applications/serial_lte_modem/src/slm_at_socket.c

```diff
@@ -904,9 +889,8 @@ static int do_recvfrom(int timeout, int flags)
 			(void)inet_ntop(AF_INET6, &((struct sockaddr_in6 *)&remote)->sin6_addr,
 			    peer_addr, sizeof(peer_addr));
 		}
-		sprintf(rsp_buf, "\r\n#XRECVFROM: %d,\"%s\"\r\n", ret, peer_addr);
-		rsp_send(rsp_buf, strlen(rsp_buf));
-		rsp_send(rx_data, ret);
+		rsp_send("\r\n#XRECVFROM: %d,\"%s\"\r\n", ret, peer_addr);
+		data_send(rx_data, ret);
```

Comment:
Would it make sense to reuse the `data_buf` for this purpose?

Reply:
Yes, just missed that in socket recv and recvfrom.
Meanwhile we continue to use buffer in stack for sending data received from AT command line, and reserve dat_buf[] for RX data only.